### PR TITLE
Incorporate standard parsers.

### PR DESCRIPTION
**Pull request by bmoyles0117**

### CODE REVIEW

**Review by qingling128 — APPROVED** [review]
The change LGTM. Is this considered a backward incompatible change?

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
Should we not bump the major version?

**Comment by igorpeshansky** [comment]
@qingling128, yes, this is a backward-incompatible change, because the log format changes, and this will change the queries you have to run to filter the logs.

**Comment by bmoyles0117** [comment]
PTAL

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
Let's also update `README.md` to indicate that major versions may introduce backward-incompatible changes and that's the latest code sends some logs as structured. We may also optionally list the structured ones...

**Comment by bmoyles0117** [comment]
I've updated the README, PTAL.

**Review by qingling128 — APPROVED** [review]
LGTM.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,12 @@ fluentd-catch-all-config
 ========================
 
 Collection of configuration files for the Fluentd log collection agent. Intended to enable the automagic collection of most logs generated by many of the most popular applications run in the cloud.
+
+Major version updates may introduce changes that are not backwards compatible.
+
+This package attempts to keep parity with the [Fluentd Parsers](https://docs.fluentd.org/v0.12/articles/parser-plugin-overview) when providing structured logs. The parsers that this package currently uses are:
```
Comment (igorpeshansky) [inline README.md]: Let's remove the `/v0.12` component from the URL.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -2,3 +2,12 @@ fluentd-catch-all-config
...
+
+* apache2
```
Comment (igorpeshansky) [inline README.md]: Minor: while we're at it, let's also use code font for these names, as they're literal parser names.

**Comment by bmoyles0117** [comment]
Addressed both comments, PTAL.

**Review by dhrupadb — APPROVED** [review]
LGTM 👌

File: README.md [hunk 3]
```diff
@@ -2,3 +2,12 @@ fluentd-catch-all-config
 ========================
 
 Collection of configuration files for the Fluentd log collection agent. Intended to enable the automagic collection of most logs generated by many of the most popular applications run in the cloud.
+
+Major version updates may introduce changes that are not backwards compatible.
```
Comment (igorpeshansky) [inline README.md]: Just realized -- let's bold this, to make sure people see this right away.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
One last thing.

**Review by qingling128 — APPROVED** [review]
LGTM.

File: README.md (hunk 3 above)
Comment (bmoyles0117) [reply README.md]: Done.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**